google_guest_agent/main.go

```diff
@@ -87,18 +87,25 @@ func closeFile(c io.Closer) {
 }
 
 func availableManagers() []manager {
+	managers := []manager{
+		&addressMgr{},
+	}
+
 	if runtime.GOOS == "windows" {
-		return []manager{
+		return append(managers,
+			&addressMgr{},
 			newWsfcManager(),
 			&winAccountsMgr{},
 			&diagnosticsMgr{},
-		}
+		)
 	}
-	return []manager{
+
+	return append(managers,
+		&addressMgr{},
```

Comment:
looks like we're appending `addressMgr` here as well. Same in case of windows

Reply:
indeed. fixed. thanks.